scripts/hiseqx/xdemuxtiles.bash

```diff
@@ -7,7 +7,7 @@ set -eu -o pipefail
 # PARAMS #
 ##########
 
-VERSION=5.0.1
+VERSION=5.2.1
```

Comment:
Crap. I had this file stashed away in another repo, copied it to this one and I forgot to update master when I branched. so this version should get resolved away once we merge back into branch. Hope that makes sense.